index.js

```diff
@@ -5,7 +5,26 @@ const OSC = '\u001B]';
 const BEL = '\u0007';
 const SEP = ';';
 
-const isTerminalApp = process.env.TERM_PROGRAM === 'Apple_Terminal';
+/* global window */
+const isBrowser = typeof window !== 'undefined' && typeof window.document !== 'undefined';
+
+let isTerminalApp_;
+let platform_ = 'browser';
```

Comment:
Just make a `isWindows` variable instead. As that's what `platform` is used for.